SAKPaaS/src/app/components/home/home.component.scss

```
@@ -30,3 +30,11 @@
     opacity: 0.3;
   }
 }
+
+@media (max-width: 600px) {
+  #home-wrapper {
+    border: none;
```

You propably want to add: 
```suggestion
    border: none;
    padding-bottom: 0;
```